lib/scanner.js

```diff
@@ -154,12 +154,33 @@ function scanQR (callback) {
   })
 }
 
-function scanPDF417 (callback) {
+function scanPDF417 (callback, idCardStillsCallback) {
+  let captureIdCardPhoto = false
+  let idCardTimer = 4000
+  let endInterval = 24000
```

Comment:
These timers could be moved to `software_config.json`. Probably is not a huge deal, it's your choice.

Reply:
I don't see `endInterval` being reassigned anywhere, so it could be changed to `const`